bindings/electron/src/index.d.ts

```diff
@@ -50,6 +50,14 @@ export enum InvitationStatus {
     Pending = 'InvitationStatusPending',
 }
 
+export enum LogLevel {
+    Debug = 'LogLevelDebug',
```

Comment:
Funny to see the autogenerated code doesn't keep the order :thinking:

Reply:
The enum members are sorted:

https://github.com/Scille/parsec-cloud/blob/c82097aa28682aeb153de4b94bd02bfb66b35658/bindings/generator/generate.py#L629-L630